docker/connector-db/sample.env

```diff
@@ -0,0 +1,7 @@
+MYSQL_ROOT_PASSWORD=unstract_root_pass
+MYSQL_DATABASE=unstract_db
+MYSQL_USER=unstract_dev
+MYSQL_PASSWORD=unstract_pass
+ACCEPT_EULA=Y
+MSSQL_SA_PASSWORD=unstractPassword@123
+MSSQL_DATABASE=master
```

Comment:
@kirtimanmishrazipstack remove this. This is misleading. I don't think using this will change actual default DB right?

Reply:
okay @ritwik-g removing 'MSSQL_DATABASE' part